libs/utils/utils.js

```diff
@@ -1002,6 +1011,7 @@ function decorateDocumentExtras() {
 }
 
 async function documentPostSectionLoading(config) {
+  window.dispatchEvent(new Event(MILO_EVENTS.POST_SECTION_LOADING));
```

Comment:
we need this event to know when to display our footer. Otherwise we're displaying it too early (leads to CLS) or too late (bad user experience)